core/src/main/java/tech/tablesaw/io/csv/CsvReadOptions.java

```diff
@@ -281,5 +281,14 @@ public Builder minimizeColumnSizes() {
       super.minimizeColumnSizes();
       return this;
     }
+
+    /**
+     * @param b if true, reads strings that appear as percentages (eg. 10%) as numbers (0.1)
+     * @return
```

Comment:
We should not leave empty @return annotations in the javadoc.

Reply:
Done.